README.md

```diff
@@ -81,6 +81,18 @@ user.tags -= ["awesome"]
 user.tags -= ["awesome", "slick"]
 ```
 
+### Validations
+
+#### `allow_list`
+
+If a allow list is defined for a tag, it will be validated to ensure that each item in tags array is allowed.
+
+```ruby
+  class User < ActiveRecord::Base
+    taggable_array :colors, allow_list: %w[red blue green]
```

Comment:
Feels like this could use a better naming. Might be that “allowed” or "accepted" sound simpler? I lean towards "allowed".

`taggable_array :colors, allowed: %w[red blue green]`

Reply:
Changed to allowed